spec/services/solidus_square/orders/retrieve_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_String_literal: true
+
+RSpec.describe SolidusSquare::Orders::Retrieve do
+  subject(:retrieve) { described_class.new(client: client, order_id: square_order_id) }
+
+  let(:client) do
+    ::Square::Client.new(
+      access_token: ENV['SQUARE_ACCESS_TOKEN'],
+      environment: "sandbox"
+    )
+  end
+  let(:square_order_id) { "aCvoi0WsmnpeNRIs7BjEHVtarh4F" }
```

Comment:
You are using an order_id that exists only on your square account. This will prevent anyone else to refresh the cassettes.
If you want to use VCR here you should also populate the sandbox account with at least one order and let it work with any account. Otherwise, it would be better to mock everything.

Reply:
https://github.com/nebulab/solidus_square/pull/36/commits/6710f713b2e595d1345eb1ec4e3cccd08f52a1b8